nemo_skills/evaluation/metrics/utils.py

```diff
@@ -43,3 +42,32 @@ def is_correct_judgement(judgement):
         return False  # improper judgement format, so have to judge as false
     verdict = judgement.split('Judgement:')[-1].strip()
     return verdict.lower() == 'yes'
+
+
+def get_metrics(metric_type: str):
+
+    from nemo_skills.evaluation.metrics.answer_judgement_metrics import AnswerJudgementMetrics
+    from nemo_skills.evaluation.metrics.arena_metrics import ArenaMetrics
+    from nemo_skills.evaluation.metrics.code_metrics import CodeMetrics
+    from nemo_skills.evaluation.metrics.if_metrics import IFMetrics
+    from nemo_skills.evaluation.metrics.lean4_metrics import Lean4Metrics
+    from nemo_skills.evaluation.metrics.math_metrics import MathMetrics
+    from nemo_skills.evaluation.metrics.mtbench_metrics import MtBenchMetrics
+
+
+    METRICS_MAP = {
```

Comment:
Please move outside of the function, so that it's possible to extend this dictionary in the downstream code

Reply:
Done